src/solver/conv_asm_implicit_gemm_gtc_bwd.cpp

```diff
@@ -30,6 +30,8 @@
 #include <miopen/gcn_asm_utils.hpp>
 #include "implicitgemm_util.hpp"
 
+MIOPEN_DECLARE_ENV_VAR(MIOPEN_DEBUG_CONV_IMPLICIT_GEMM_ASM_BWD_GTC)
```

Comment:
'MIOPEN_DEBUG_CONV_IMPLICIT_GEMM_ASM_BWD_GTC_XDLOPS' shuold be better, to keep same naming convention with fwd/wrw

Reply:
fixed